aioelasticsearch/transport.py

```diff
@@ -252,6 +252,11 @@ async def _perform_request(
             connection = await self.get_connection()
 
             try:
+                # add a delay before attempting the next retry
+                # 0, 1, 3, 7, etc...
+                delay = 2**(attempt - 1) - 1
```

Comment:
I believe the upper cap is required, otherwise the delay may grow to insane values